### PR TITLE
Clear createpipelinerun when x is pressed on pipeline

### LINKED ISSUE #1395: Pressing `x` on Pipeline in Create Pipelinerun should clear the page
# Expected behavior
It clears the pipeline related options below 

# Actual behavior
It doesnt 

# Steps to reproduce the problem

1.
2.
3.

# Environment
- Kubernetes Platform:

- Tekton Pipelines version:

- Tekton Triggers version:

- Tekton Dashboard version:

# Additional Info

### PR DESCRIPTION
**Pull request by CarolynMabbott**
# Changes
Fixes #1395 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/CreatePipelineRun/CreatePipelineRun.js [hunk 1]
```diff
@@ -273,6 +273,16 @@ class CreatePipelineRun extends React.Component {
...
       });
+    } else {
+      // Reset pipelineresources and params when no Pipeline is selected
+      this.setState({
+        ...initialPipelineInfoState(),
+        [NAMESPACE]: this.getPipelineInfo('namespace'),
+        [PIPELINE_REF]: '',
+        [RESOURCE_SPECS]: [],
+        [PARAM_SPECS]: [],
+        [PIPELINE_ERROR]: false
```
Comment (AlanGreene) [inline src/containers/CreatePipelineRun/CreatePipelineRun.js:284]: `initialPipelineInfoState` already returns these values, so the only one we need to keep here is `[NAMESPACE]`

Comment (CarolynMabbott) [reply src/containers/CreatePipelineRun/CreatePipelineRun.js]: Have removed these now

**Pull request merged**